### ISSUE TITLE
Bubble menu popovers don't work on Safari

### ISSUE THREAD

**Issue by OtisTemler** [hunghg255/reactjs-tiptap-editor#126]
On Safari (18.1.1), it doesn't seem to be possible to open the link or the color popovers, see attached video from the official demo page:

https://github.com/user-attachments/assets/9b9dc400-f8a4-4916-a850-ef83b206acc6

Chrome works all fine.

**Comment by hunghg255** [comment]
![Screenshot 2024-12-19 at 23 07 00](https://github.co@OtisTemler /user-attachments/assets/6e1a52bd-f84b-4b12-b1aa-4b8fcc5882df)
LOL wth, it seems appear and hidden at corner

**Comment by OtisTemler** [comment]
Yeah, super weird... And if you apply `onOpenAutoFocus={(e) => e.preventDefault()}` to `PopoverContent` it does stay open, but whenever you interact with the content it's gone again.

**Comment by hunghg255** [comment]
> >

oke. i will check later. i think the issue related to shadcn or radix ui
